src/TypeScript/Analyzer/JavaScriptAnalyzer.cs

```diff
@@ -41,31 +41,27 @@ internal sealed class JavaScriptAnalyzer : IAnalyzer, IDisposable
     {
         private readonly EventWaitHandle serverInitLocker = new EventWaitHandle(true, EventResetMode.AutoReset);
 
-        private readonly IEslintBridgeClientFactory eslintBridgeClientFactory;
```

Comment:
As we discussed, the IEslintBridgeProcess is now removed from the analyzer. The analyzer is only aware of the Client now.